project/src/main/java/com/google/sps/utilities/DatabaseWrapper.java

```diff
@@ -82,4 +64,9 @@ public void updateUser(User user) {
     mutations.add(builder.build());
     return mutations;
   }
+
+  /** Close the Spanner database connection. */
```

Comment:
Just curious, if one servlet close the Spanner, does it affect other Servlets to use the Wrapper?

Reply:
After checking documentation, this will only close the spanner instance created in the constructor in DatabaseService. Each servlet has its own spanner instance as it will call new DatabaseWrapper(new DatabaseService()).